nion/swift/Inspector.py

```diff
@@ -4985,3 +4975,60 @@ def data_item_delete() -> None:
             return widget
 
         return None
+
+
+def make_line_plot_display_layer_button_row(document_controller: DocumentController.DocumentController,
```

Comment:
Double blank line before method.

Reply:
This is not a class method, so I think 2 blank lines is correct.